packages/runtime/src/components/hooks/useMediaQuery.ts

```diff
@@ -1,6 +1,4 @@
-import { useSyncExternalStore } from 'use-sync-external-store/shim'
```

Comment:
Here and below, migrate stale imports of `useSyncExternalStore` from `use-sync-external-store/shim` to [React 18's native implementation](https://github.com/reactwg/react-18/discussions/86#discussioncomment-1567159). 

(We already do that in most of the new code).